repose-aggregator/components/filters/rate-limiting-filter/src/test/java/org/openrepose/filters/ratelimiting/SchemaTest.java

```diff
@@ -87,19 +87,5 @@ public void shouldValidateAgainstLiveLimitsExample() throws Exception {
             assertThat("Expected element should have child elements",
                     jaxbUnmarshaller.unmarshal(sampleSource, Limits.class).getValue().getRates().getRate().size(), greaterThan(0));
         }
-
-        @Test
-        @Ignore
-        public void output() throws Exception {
```

Comment:
Everything still seemed to work. I would let it run through the full integration tests, of course.